pkg/agent/agent_test.go

```diff
@@ -188,24 +206,22 @@ func TestListReleases(t *testing.T) {
 				t.Errorf("got: %d, want: %d", got, want)
 			}
 
-			//Check if expected and returned contain the same releases, given they have same size
-			m := make(map[releaseStub]bool)
-
-			for _, eapp := range tc.expectedApps {
-				m[releaseStub{eapp.ReleaseName, eapp.Namespace, strtoi(eapp.Version)}] = true
-			}
-
-			for _, app := range apps {
-				rs := releaseStub{app.ReleaseName, app.Namespace, strtoi(app.Version)}
-				if _, ok := m[rs]; !ok {
-					t.Errorf("got: %v, want: %v", rs, "None")
+			//Deep equality check of expected aginst attained result
+			for i := 0; i < len(apps); i++ {
+				if !cmp.Equal(apps[i], tc.expectedApps[i]) {
+					t.Errorf(cmp.Diff(apps[i], tc.expectedApps[i]))
 				}
 			}
 
 		})
 	}
 }
 
+// getAppIdentity concatenates the release name and namespace of an app
+func getAppIdentity(app proxy.AppOverview) string {
+	return fmt.Sprintf("%v %v", app.ReleaseName, app.Namespace)
+}
```

Comment:
This looks like it was left over from some testing or earlier sorting? I don't see it being used.